File: .analysis/check_braces.js
const fs = require('fs');
const path = 'c:\\Users\\WALTON\\Documents\\hishab_pro_new\\lib\\main.dart';
const text = fs.readFileSync(path,'utf8');
['{','}','(',')','[',']'].forEach(ch=>console.log(ch+':', (text.split(ch).length-1)));
const lines = text.split('\n');
console.log('\n---TAIL 60 LINES---');
lines.slice(-60).forEach((l, i)=>console.log((lines.length-60+i+1).toString().padStart(5)+': '+l));

File: .analysis/check_braces.py
path = r"c:\Users\WALTON\Documents\hishab_pro_new\lib\main.dart"
with open(path, 'r', encoding='utf-8') as f:
    text = f.read()
for ch in ['{','}','(',')','[',']']:
    print(ch, text.count(ch))
# print last 50 lines for quick context
lines = text.splitlines()
print('\n---TAIL 60 LINES---')
for l in lines[-60:]:
    print(l)
